src/plugins/converter/converter.plugin.ts

```diff
@@ -0,0 +1,39 @@
+import { SpotterOption, SpotterPlugin, SpotterPluginLifecycle } from '../../core';
+import icon from './icon.png';
+export class ConverterPlugin extends SpotterPlugin implements SpotterPluginLifecycle {
+
+  identifier = 'Converter';
+
+  async onQuery(query: string): Promise<SpotterOption[]> {
+       
+    try {
+       const splitted = query.split(" ",4)
```

Comment:
`1euro in USD` without space won't work. I think we need a regexp here or so on.